packages/transactions/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -70,19 +87,39 @@ export const OutboundShipmentListView: FC = () => {
   ];
 
   const tableApi = useDataTableApi<Transaction>();
+  const t = useTranslation();
 
   return (
     <>
       <AppBarContentPortal>
-        <Dropdown label="Select" value={10} IconComponent={Customers}>
-          <DropdownItem value={10}>Ten</DropdownItem>
-          <DropdownItem value={20}>Twenty</DropdownItem>
+        <Dropdown label="Select">
+          <DropdownItem
+            IconComponent={ChevronDown}
+            onClick={() => {
+              const linesToDelete = tableApi?.current?.selectedRows;
+              if (linesToDelete && linesToDelete?.length > 0) {
+                mutateAsync(linesToDelete);
+                success(`Deleted ${linesToDelete?.length} invoices`)();
```

Comment:
Bit weird actually? Maybe we should return 

```
const success = (message:string) => enqueueSnackbar(message, {variant: 'success'})
```

rather than the function? 

I admit I was a bit silly and got a bit confused passing `onClick={() => success("..")}`  - seems like the most common usage

Reply:
probably should - the function was a convenience. which only seems to help in that one specific scenario.

If we do keep the function, then this might make this usage clearer:

```
                const notifyDeleteSuccess = success(`Deleted ${linesToDelete?.length} invoices`);

// and then, in the delete event:
                notifyDeleteSuccess();
```